packages/wallet/src/utils/transaction-generator.ts

```diff
@@ -126,3 +138,22 @@ export function createDepositTransaction(destination: string, depositAmount: str
   }
   return createNitroDepositTransaction(destination, expectedHeld, depositAmount);
 }
+
+export function createTransferAllTransaction(source: string, destination: string, amount: string) {
+  const allocation: Allocation = [
+    {
+      destination: normalizeDestinationAddress(destination),
+      amount
+    } as AllocationItem
+  ];
+  return createNitroTransferAllTransaction(source, allocation);
+}
+
+function normalizeDestinationAddress(address: string): string {
```

Comment:
What does "normalize" mean? Maybe this should be more explicit and say that it makes any address or external address into an external address.

Reply:
Yeah, perhaps `externalizeAddress`